File: src/main/intellibitz/sted/actions/SelectAllAction.java
package intellibitz.sted.actions;

import javax.swing.event.ListSelectionEvent;
import java.awt.event.ActionEvent;

public class SelectAllAction
        extends TableModelListenerAction {
    public SelectAllAction() {
        super();
    }


    public void valueChanged(ListSelectionEvent e) {
    }

    public void actionPerformed(ActionEvent e) {
        super.selectAll();
        fireStatusPosted("Selected All");
    }

}

File: src/main/intellibitz/sted/actions/SelectAllAction.kt
package sted.actions

import java.awt.event.ActionEvent
import javax.swing.event.ListSelectionEvent

class SelectAllAction : TableModelListenerAction() {
    override fun valueChanged(e: ListSelectionEvent) {}
    override fun actionPerformed(e: ActionEvent) {
        super.selectAll()
        fireStatusPosted("Selected All")
    }
}